contracts/mixnet/src/transactions.rs

```diff
@@ -66,10 +66,9 @@ pub(crate) fn try_remove_mixnode(
     env: Env,
 ) -> Result<HandleResponse, ContractError> {
     // find the bond, return ContractError::MixNodeBondNotFound if it doesn't exist
-    let mixnode_bond = match mixnodes_read(deps.storage).may_load(info.sender.as_bytes())? {
-        None => return Err(ContractError::MixNodeBondNotFound {}),
-        Some(bond) => bond,
-    };
+    let mixnode_bond = mixnodes_read(deps.storage)
+        .may_load(info.sender.as_bytes())?
+        .ok_or_else(|| ContractError::MixNodeBondNotFound {})?;
```

Comment:
Could you change it to `.ok_or(ContractError::MixNodeBondNotFound {})?;` to satisfy https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_lazy_evaluations ?